boards/PDM/Src/Io/Io_Efuse.c

```diff
@@ -0,0 +1,231 @@
+#include <assert.h>
+#include <math.h>
+#include "Io_Efuse.h"
+#include "configs/Io_EfuseConfig.h"
+
+#pragma GCC diagnostic ignored "-Wconversion"
+
+struct Efuse_Context
+{
+    // The SPI handle for the SPI device the E-Fuse is connected to
+    SPI_HandleTypeDef *efuse_spi_handle;
+
+    // The current state of the watchdog-in bit (bit 15). If the watchdog is
+    // enabled its state must be alternated at least once within the watchdog
+    // timeout period.
+    bool wdin_bit_to_set;
+};
+
+struct Efuse_Context aux1_aux2_efuse;
```

Comment:
Currently this structure needs to be accessed by both _Io_Efuse.c_ here and _Io_Efuses.c_ [here](https://github.com/UBCFormulaElectric/Consolidated-Firmware/blob/e68557881126b3328ca0a1ebb7f28fe656b4f5a3/boards/PDM/Src/Io/Io_Efuses.c#L72).
`struct Efuse_Context aux1_aux2_efuse;` currently is not in  _Io_Efuses.c_ scope. What is the best way to fix this linking issue?

Reply:
Should have a `struct Efuses` in `Io_Efuses.c` with members consisting of pointers to `struct Efuse_Context` for each efuse (assuming we go with an Io_Efuses containing all the efuses).